### DISCUSSION TITLE
多签钱包小问题

### DISCUSSION THREAD

**Discussion by nft-maker-one** [AmazingAng/WTF-Solidity#507] [General]
原教材这个地方有个小错误，success没有定义就直接赋值了
![image](https://github.com/AmazingAng/WTF-Solidity/assets/121859606/0d60bd61-7442-4a4c-ac1e-5876af2b76e8)
应该是
![image](https://github.com/AmazingAng/WTF-Solidity/assets/121859606/3cf47f24-1a13-4a51-9251-648c2d386dde)

没有错误，纯我眼瞎，上面定义了,大家写的时候也别写漏了
![image](https://github.com/AmazingAng/WTF-Solidity/assets/121859606/ffd5120f-793e-42bb-86eb-f10130c48161)

**Comment by AmazingAng** [comment]
在 returns 里定义了:
```  
  function execTransaction(
        address to,
        uint256 value,
        bytes memory data,
        bytes memory signatures
    ) public payable virtual returns (bool success) {
```

**Reply by nft-maker-one** [reply]
不好意思大佬，纯我眼瞎了，抱歉抱歉
